protos/feast/core/CoreService.proto

```diff
@@ -202,6 +269,19 @@ message ListStoresResponse {
     repeated feast.core.Store store = 1;
 }
 
+message ApplyEntityRequest {
+    // If project is unspecified, will default to 'default' project.
+    // If project specified does not exist, the project would be automatically created.
+    feast.core.Entity entity = 1;
+
+    // Name of project that this entity belongs to.
+    string project = 2;
+}
+
+message ApplyEntityResponse {
+    feast.core.Entity entity = 1;
```

Comment:
Not sure if returning the entity would be useful.
Would returning a status be useful instead?

Reply:
Not too sure what you mean here. Aren't we getting rid of the use of Status?

Reply:
Hmm ok that I will get rid of statuses on my end as well.